src/getTreeColumnWidthFromArgs.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Extracts the tuple value from the tree-column-width argument value.
+ * The tuple is in the shape of [baseColumnWidth, levelDepthMultiplier]
+ */
+export function getTreeColumnWidthFromArgs(
+  argString: string,
+): [number, number] {
+  const values = argString
+    .split(",")
+    .map((stringValue) => parseInt(stringValue));
+
+  if (values.length !== 2) {
+    throw new Error(
+      "The value passed into the column-width argument must be two numbers separated by a comma",
+    );
+  }
+
+  const [baseWidth, levelColumnMultiplier] = values;
```

Comment:
Let's align the name with the one you give it in the multiline comment above the function.

```suggestion
  const [baseWidth, levelDepthMultiplier] = values;
```